e2e/testdata/fn-render/path-index-ancestor/db/Kptfile

```diff
@@ -0,0 +1,8 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: db
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/starlark:unstable
```

Comment:
Don't use unstable in kpt e2e test

Reply:
What should I use instead? Unstable is the tag used in the already existing tests and I don't see another tag available in gcr.io/kpt-fn/starlark

Reply:
It is fine to use unstable for now. We will do a sweeping change in all the tests to use versioned images post m1 and I hope to have all the functions also tagged by then.